bot.py

```diff
@@ -700,11 +700,215 @@ def cmd_shoutout(match: Match[str]) -> Response:
     )
 
 
+class YouTubeVideo(NamedTuple):
+    video_id: str
+    title: str
+    playlist_id: str
+    playlist_position: int
+
+    def chat_message(self) -> str:
+        return (
+            f'{self.title} - '
```

Comment:
the title should be `esc()`aped here (otherwise a curly brace in the title could break the bot or leak the creds) -- and yeah I trust myself to not do this in a title but it's better to be thorough